allowances/contracts/SignatureDecoder.sol

```diff
@@ -0,0 +1,54 @@
+// SPDX-License-Identifier: LGPL-3.0-only
```

Comment:
This module cannot be imported from the Safe Contracts package?

Reply:
no, because the safe-contracts run solidity 0.5.x and this one runs 0.7.x

I didn't want to skip 2 major version for this :/